src/components/pages/Favorites/Component.js

```diff
@@ -1,9 +1,24 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Canvas } from '../../layouts';
 import SearchBar from '../../common/SearchBar';
+import CityCard from '../../common/CityCard/Component.js';
 import '../../../styles/ComparisonPage.less';
+import axios from 'axios';
 
-function favtitle() {
+//156, 320, 747, 1177
```

Comment:
?? What are these numbers? Please add a description as to why this is commented out, otherwise delete it please.